.github/workflows/buildAndDeploy.yml

```
@@ -1,11 +1,9 @@
 name: Build and Deploy
 
 on:
-  pull_request:
-    branches: [main, develop]
-    types: [closed]
   push:
-    branches: [main, develop]
+    branch:
```

actionlint (1.7.7)

unexpected key "branch" for "push" section. expected one of "branches", "branches-ignore", "paths", "paths-ignore", "tags", "tags-ignore", "types", "workflows"

(syntax-check)